sdk/basyx/object_store.py

```diff
@@ -17,7 +17,6 @@
 
 # We define types for :class:`aas_core3.types.Identifier` and :class:`aas_core3.types.Referable` for easier referencing.
 _IdentifiableType = TypeVar('_IdentifiableType', bound=Identifiable)
-_ReferableType = TypeVar('_ReferableType', bound=Referable)
```

Comment:
Why was this removed? I guess it was not used?

Reply:
Yes it was unused